addons/l10n_din5008/report/din5008_report.xml

```diff
@@ -103,8 +103,12 @@
                                 <t t-foreach="o.l10n_din5008_addresses" t-as="doc_address">
                                     <td>
                                         <div class="shipping_address">
-                                            <strong><t t-esc="doc_address[0]"/></strong>
-                                            <div t-esc="doc_address[1]" t-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": True}'/>
+                                            <strong><t t-esc="doc_address.get('title')"/></strong>
```

Comment:
yeah this is breaking the stable, we will do the simple fix you did last time since in 17.4 the code change it will be there for only 17.0 and 17.2 so i think it's ok, sorryyyyy :grimacing:

Reply:
Maybe it's a small refactoring I could do on master